GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -45,10 +45,10 @@ typedef NS_ERROR_ENUM(kGIDSignInErrorDomain, GIDSignInErrorCode) {
   kGIDSignInErrorCodeCanceled = -5,
   /// Indicates an Enterprise Mobility Management related error has occurred.
   kGIDSignInErrorCodeEMM = -6,
-  /// Indicates there is no `currentUser`.
-  kGIDSignInErrorCodeNoCurrentUser = -7,
   /// Indicates the requested scopes have already been granted to the `currentUser`.
   kGIDSignInErrorCodeScopesAlreadyGranted = -8,
+  /// Indicates there is an operation on a previous user.
+  kGIDSignInErrorCodePreviousUser = -9,
```

Comment:
It's not clear to me what "previous user" means as an error without reading the comment. I'd suggest a more descriptive name such as "kGIDSignInErrorCodeMismatchWithCurrentUser".

Reply:
Done.